### ISSUE TITLE
Errors only show up after I change tabs

### ISSUE THREAD

**Issue by sillyotter** [fsprojects/atom-fsharp#36]
If I open up a fsharp project and purposefully break something and save it, I see no errors.  if i switch to a different buffer, open a new file, etc, an then go back, the error shows up.  

This is with atom 0.198.0 on windows 8 and the latest fsharp packages installed as per the readme.

Let me know if I can provide any more information.

**Comment by Krzysztof-Cieslak** [comment]
Does everything is working correctly after You go back to original file? I mean, do You get tooltips, autocomplete and errors are showing / hiding as You write code?

Looks like it's some problem with activation of plugin at start of Atom. Do You have any other plugins (different then required by this plugin) installed?

Anyway, error highlighting should be upgraded Soon™ to be more responisve

**Comment by Krzysztof-Cieslak** [comment]
Also, does folder You open with Atom contains `fsproj` file or is it just a standalone F# file?

**Comment by Krzysztof-Cieslak** [comment]
@sillyotter, Can You try `0.1.9` version and check if problem still occurs?

**Comment by sillyotter** [comment]
Sorry for the delay.

I just installed the updated versions.  There is nothing else installed on this machine, plugin wise. The folder I opened had a .sln in it, and a folder that had both .fs and .fsproj files in it.  I was editing one of the .fs files in that nested folder.

Let me describe what I'm seeing, perhaps it will make sense.  I fired up atom, opened my folder, navigated to and opened my file.

I let a decent interval pass to allow things to parse  No idea if this is necessary, but i know that vim takes a while for the tooling to catch up.  About two minutes this time as I was typing this up.  I'm editing line 17 in the images below, backspacing over and retyping `toList`

![start](https://cloud.githubusercontent.com/assets/483712/7551467/3034e5cc-f657-11e4-904e-611cd7003103.PNG)

I can hover over functions and see information about it, but then I start to back space over a function call.  Nothing is show indicating that the new code is wrong.

![noerror](https://cloud.githubusercontent.com/assets/483712/7551469/3cbca2d0-f657-11e4-8ec0-eb96f7535b9a.PNG)

I type a character, the popup shows up suggesting a completion, and an error shows up indicating that my new partial method doesn't exist.

![error](https://cloud.githubusercontent.com/assets/483712/7551470/5e9e9764-f657-11e4-9cbd-6013b103cb32.PNG)

If I type it all the way out, via keyboard, the error goes away, but the popup is still there.  I hit tab to make the popup go away.

If I backspace over the text, no error shows up.

![newnoerror](https://cloud.githubusercontent.com/assets/483712/7551473/64a9f68a-f657-11e4-973d-10598c5b4a11.png)

If I then start to type, an auto complete prompt and error shows up.

If I hit tab to accept the completion, the function is entered, but the error stays there.

![aftercomplete](https://cloud.githubusercontent.com/assets/483712/7551476/7858c08a-f657-11e4-9385-e2f9983ed91a.PNG)

**Comment by Krzysztof-Cieslak** [comment]
OK, as for moment I see 2 problems:

Unfortunately right now plugin does not support `sln` files and assumes that `fsproj` file is in root folder of Atom. Please try opening sub-folder with `fsproj` file as a root folder for Atom.

Second point - Have You done following "install "autocomplete-plus" package (disable build-in atom autocomplete for best experience)" ?

**Comment by sillyotter** [comment]
I followed the install instructions exactly.  I disabled the autocomplete package that is built in and installed the autocomplete-plus version.  autocomplete-plus version 2.13.1, FSharp 0.1.9, and language-fsharp 0.8.4 is whats there.

Also, I just re-tested things after opening the project folder directly.  Everything behaved exactly as I reported above.  No difference at all.  

Wish I could be of more help.  I looked and saw that atom is reporting two deprecated calls in the fsharp package.  

![errors-fsharppng](https://cloud.githubusercontent.com/assets/483712/7551745/73aa7064-f662-11e4-9714-584e4378ef4f.PNG)
